src/utils/hooks.ts

```diff
@@ -13,8 +13,7 @@ const useIsLoggedIn = (): boolean | undefined => {
   const { token } = useContext(TokenContext);
   // application is still loading
   if (token === undefined) return undefined;
-  const tokenValid = typeof token === "string" ? token.length > 0 : false;
-  return tokenValid;
+  return typeof token === "string" ? token.length > 0 : false;
```

Comment:
I purposely introduced that `tokenValid` variable to give it some semantic

Reply:
We can a comment instead.

I did it because it is useless to have a variable just returned after.